openwisp-config/files/lib/openwisp/net.lua

```diff
@@ -3,8 +3,8 @@ local uci = require('uci')
 local net = {}
 
 function net.get_interface(name, family)
+    family = family or 'inet'
```

Comment:
@okraits do you understand why this change is needed? I don't.

Reply:
@nemesisdesign I know that there was a warning for the previous version of this line. But indeed, the new version of this line seems more ambiguous to me than the previous version.
I would prefer: 

    local ip_family = family or 'inet'

and further down then

    if interface.name == ifname and interface.family == ip_family then